backend/dataall/modules/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""
+Contains all submodules that can be plugged into the main functionality
+
+How to migrate to a new module:
+1) Create your python module
+2) Create the following submodules:
+    a) gql
+    b) tasks
+    3) cdk
```

Comment:
[Question] - @nikpodsh IMPORTANT! something that we have not considered yet are other long-running tasks besides the cdk task. There are 7 ECS tasks, from which 2 run on-demand (cdkproxy and share-manager), the other 5 run on different schedules:
![image](https://user-images.githubusercontent.com/71252798/225567665-e181596a-973b-4378-a972-8dc2d0e098c4.png)

I am not sure about the utility of some of these scheduled tasks (e.g. subscriptions seems to be a legacy task). I can review which ones we really need. I can confirm that cdkproxy, share-manager, stack-updater, catalog-indexer and table-syncer should be included

Reply:
They don't need loading of modules as far as I see. They are either isolated or use small portion of the code that they load explicitly. I think most of them will live under core, but something like sharing can be moved to modules (under cdk or we can add another submodule just for them).